PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -141,6 +148,25 @@ def nanoAOD_addTauIds(process):
                                    process.rerunMvaIsolationSequence)
     return process
 
+def nanoAOD_addBoostedTauIds(process):
+    updatedBoostedTauName = "slimmedTausBoostedNewID"
+    boostedTauIdEmbedder = tauIdConfig.TauIDEmbedder(process, debug=False, 
+                                                     originalTauName = "slimmedTausBoosted",
+                                                     updatedTauName = updatedBoostedTauName,
+                                                     postfix="Boosted",
+                                                     toKeep = [ "2017v2", "dR0p32017v2", "newDM2017v2","againstEle2018",])
+    boostedTauIdEmbedder.runTauID()
+    process.boostedTauSequence.insert(process.boostedTauSequence.index(getattr(process, "finalBoostedTaus")),
+                                      process.rerunMvaIsolationSequenceBoosted)
+
+    process.boostedTauSequence.insert(process.boostedTauSequence.index(getattr(process, "finalBoostedTaus")),
+                                      getattr(process, updatedBoostedTauName))
```

Comment:
Couldn't this be done as for standard taus above? Btw. in the current case you don't need the getattr

Reply:
You mean with only inserting the `rerunMvaIsolationSequenceBoosted`? I tried that originally, but I believe this set-up is required. The above uses the tool to add to an already existing Tau ID task/sequence, but for boosted taus this does not exist, so it has to be added explicitly by the second. Or at least, that is my recollection of how this works.

Reply:
Ok I see. Thanks for clarification. Indeed there is an existing ID sequence for std taus.